### ISSUE TITLE
Symfony serving with Default PHP on Mac OS 

### ISSUE THREAD

**Issue by prondubuisi** [dkarlovi/opentelemetry-php-user-test#3]
Hello @dkarlovi I am having this error while loading the homepage
Fatal Error: composer.lock was created for PHP version 7.4 or higher but the current PHP version is 7.3.24.
<img width="1676" alt="Composer conflict" src="https://user-images.githubusercontent.com/22311928/106391626-9d752b00-63ee-11eb-9283-46f73b8fdb6e.png">

I also noticed that when I run `symfony serve` it  loads with PHP 7.3.24 

```
Ndubuisis-MacBook-Pro:otel-php-test prondubuisi$ symfony serve
                                                                                                                        
 [WARNING] run "symfony server:ca:install" first if you want to run the web server with TLS support, or use "--no-tls"  
  to avoid this warning                                                                                                 
                                                                                                                        

Tailing Web Server log file (/Users/prondubuisi/.symfony/log/bbc0d23ccc389163a9709966033259f8506d4bc7.log)
Tailing PHP-FPM log file (/Users/prondubuisi/.symfony/log/bbc0d23ccc389163a9709966033259f8506d4bc7/53fb8ec204547646acb3461995e4da5a54cc7575.log)
                                                                                                                        
 [OK] Web server listening                                                                                              
      The Web server is using PHP FPM 7.3.24                                                                            
      http://127.0.0.1:8000                                                                                             
                                                                                                                        

[Web Server ] Jan 31 17:45:36 |DEBUG  | PHP    Reloading PHP versions 
[Web Server ] Jan 31 17:45:37 |DEBUG  | PHP    Using PHP version 7.3.24 (from default version in $PATH) 
[Web Server ] Jan 31 17:45:37 |INFO   | PHP    listening path="/usr/sbin/php-fpm" php="7.3.24" port=63745
[PHP-FPM    ] Jan 31 17:45:37 |NOTICE | FPM    fpm is running, pid 65095 
[PHP-FPM    ] Jan 31 17:45:37 |NOTICE | FPM    ready to handle connections 
[Web Server ] Jan 31 17:46:01 |INFO   | SERVER GET  (200) /hello ip="127.0.0.1"
[Web Server ] Jan 31 17:46:01 |INFO   | SERVER GET  (200) /favicon.ico 
[Web Server ] Jan 31 17:46:04 |INFO   | SERVER GET  (200) /hello 
[Web Server ] Jan 31 17:46:04 |INFO   | SERVER GET  (200) /favicon.ico 
^C[PHP-FPM    ] Jan 31 17:46:36 |NOTICE | FPM    Terminating ... 
[PHP-FPM    ] Jan 31 17:46:36 |NOTICE | FPM    exiting, bye-bye! 
Shut down, bye!


```

Is there a way to specify PHP  path while serving symphony?

**Comment by dkarlovi** [comment]
This seems like your local environment issue, please consult this:
https://symfony.com/doc/current/setup/symfony_server.html

**Comment by prondubuisi** [comment]
Thanks @dkarlovi fixed, my preferred PHP version is `7.4.1`, and running `echo 7.4.1 > .php-version` then `symfony serve` helped me solve this issue.

**Comment by dkarlovi** [comment]
OK. Closing here then.
